src/cli_plugin/install/settings.js

```diff
@@ -36,11 +36,15 @@ import expiry from 'expiry-js';
 
 import { fromRoot } from '../../core/server/utils';
 
+const LATEST_PLUGIN_BASE_URL =
+  'https://ci.opensearch.org/ci/dbc/distribution-build-opensearch-dashboards';
+
 function generateUrls({ version, plugin }) {
-  return [
-    plugin,
-    `https://artifacts.opensearch.org/downloads/kibana-plugins/${plugin}/${plugin}-${version}.zip`,
-  ];
+  return [plugin, generatePluginUrl(version, plugin)];
+}
+
+function generatePluginUrl(version, plugin) {
+  return `${LATEST_PLUGIN_BASE_URL}/${version}/latest/${process.platform}/${process.arch}/builds/opensearch-dashboards/plugins/${plugin}-${version}.zip`;
```

Comment:
There are a few different values for `platform` and `arch`.  I'm not sure of how many permutations of platforms and arch's we support, but should we show some sort of warning for permutations that we dont have distributions for?

Reply:
Great catch!

Updated!